### ISSUE TITLE
open chrome suddenly

### ISSUE THREAD

**Issue by KazutoRito** [deanoemcke/thegreatsuspender#938]
Hi Dev, I have a problem when I leave another tab which I setting to suspend when it's idle after 1 minute, the tab is suspended but it's not a problem. The PROBLEM is when I leave another tab like before BUT minimize / change to another ANY window app, when it's suspending auto in the background SUDDENLY chrome is showing up and it's disturb me. 

OS: Windows 10 Pro ver 1809 
HW: Intel i7 7700HQ, Nvidia GTX1050 
Chrome ver: latest
Ext. ver: v7.0.109

**Comment by ossilator** [comment]
interesting, i'm seeing something similar for some weeks/months now, but i didn't associate it with suspension - it seems more that moving the mouse on some pages (presumably causing a mouse-over event or something to that tune), which somehow makes an apparently unrelated chrome window pop to the front. note that i'm on linux/x11/kde.

**Comment by foomanfish** [comment]
I'm having the same issue as well.  It's been going on for about 3 weeks.  I'm fairly confident it's correlated with great suspender.  I just figured it out this morning, that whenever a page is suspended it'll bring chrome back up over whatever program I'm using.

I'm using Windows 10 64-bit, and the latest Chrome version, updated yesterday I believe.

**Comment by daniel-asilva** [comment]
> I'm having the same issue as well. It's been going on for about 3 weeks. I'm fairly confident it's correlated with great suspender. I just figured it out this morning, that whenever a page is suspended it'll bring chrome back up over whatever program I'm using.
> 
> I'm using Windows 10 64-bit, and the latest Chrome version, updated yesterday I believe.

Sure it is. I've been observing this exact same issue about some months ago. Always when a tab is suspended the Chrome window with that tab pops-up on top of all the others. Maybe some option on Chrome to stop this behavior?

Windows 10 1803 x64
Chrome v74.0.3729.157
TGS v7.0.109

**Comment by jay** [comment]
I'm also having a similar problem but I don't know if it has to do with the extension. It is very rare and I can't reproduce. The last two times it happened the circumstances were very similar, I hit CTRL+F to search for some text in a tab and then I started typing in the text and then a different Chrome window would pop up. It's possible the CTRL key is sticking so I've started monitoring with low level keyboard and mouse hooks to rule out any hardware issue. However I have also seen it happen under different circumstances, for example I switch to a tab with a mouse click and a different Chrome window pops up.

Windows 8.1 x64
Chrome 74.0.3729.169 (Official Build) (64-bit)
The Great Suspender 7.0.109

My options are the default except:
Automatic Tab Suspension > 'Automatically suspend tabs after:' **30 minutes**
Automatic Tab Suspension > 'Never suspend active tab in each window' **unchecked**
Suspended Tabs > 'Automatically unsuspend tab when it is viewed' **checked**
Other > 'Instantly suspend when system memory gets very low' **unchecked**

**Comment by daniel-asilva** [comment]
> It's possible the CTRL key is sticking

Hey jay. I'm absolutely sure that is not a keyboard key or a mouse behavior.

I'm a little bit crazy with tabs usage 🤣 (see below)

![2C3FF627-F2D6-4B56-937C-CA46E8CB55C1](https://user-images.githubusercontent.com/35084392/59269846-57ba8c00-8c26-11e9-8396-0a4dbccbd967.jpeg)

I tell you why I disagree:
When I'm doing some web research (almost everyday), I suffer from this behavior. And I've observed that when the tab suspension time I set it up comes up, the tab that need to be suspended is causing this behavior to Chrome window containing that tab to suddenly pops up in front of whatever program is open.

I really don't know what may have been changed on Chrome or on the Extension in the last times, 'cause this never happened in the past. It's very recent.

I'm pretty sure about this because when the tabs is suspended by the TGS it seems to load something. That loading circle on the favicon rapidly appears and it loads that fancy TGS page, and also the favicon turns dimmed.

Yesterday I noted this very annoying behavior again on another research I was doing. I oppened some pages in sequence from the Google page, and ended using just the last one I opened. After passed the given time to suspend those tabs, I was using another window and the window (containing those tabs early oppened) popped up, and one tab after another was being reloaded with the TGS page and the favicon turning dimmed. Instinctively I alt tabbed back to the window I was reading a text but, after I switched, the window pops up again until all those tabs on that window has been fineshed suspending with that same time between each one I've opened it.

It's a little bit long story, but I really hope that this can help the devs to fix it 😪
It seems to be something related with the tab not being suspended in background that is causing the Chrome window to appears in front of everything. I've not yet tested it on another computer, 'cause I also use some modified flags for Chrome on this machine and it might have some finger in this.

**Comment by jay** [comment]
@daniel-asilva do you have any of the options set that I have set?

**Comment by daniel-asilva** [comment]
> @daniel-asilva do you have any of the options set that I have set?

In the same order you presented:
1 hour
checked
unchecked
checked

**Comment by jay** [comment]
Ok so they're all different. I guess then it is not the options set. Searching through open issues I see similar #947, #951 which led me to #865 which this issue is probably a dupe of.

**Comment by daniel-asilva** [comment]
Well noted @jay. And 865 it's a long thread about it. It made me dig up into some messages with a friend when I first discovered the TGS, and a curious point about it is that it was exactly at the same month of that issue.

I'll take a look at [this branch](https://github.com/deanoemcke/thegreatsuspender/issues/865#issuecomment-470758040) (also [this](https://github.com/deanoemcke/thegreatsuspender/issues/865#issuecomment-475571313)) and do some more tests here.

And [this](https://github.com/deanoemcke/thegreatsuspender/issues/865#issuecomment-470150737) also confirms me that my guess was not wrong: it was a (security) change made on Chrome. But not that recent as a noticed this behavior on TGS.
Thanks for your time. Maybe I used the wrong keywords to look for other issues haha

**Comment by jay** [comment]
After reading #865 I checked the web store and indeed there is a newer version of the extension, 7.1.0. I did some debugging and discovered the check done by Chrome to update the extension was receiving a "noupdate" reply from Google for my installed version 7.0.109. I had to uninstall the extension and then reinstall it from the Chrome web store. I've [reported it](https://bugs.chromium.org/p/chromium/issues/detail?id=994057) to Google. I don't know whether it will fix the problem but I'm sure I'll have an idea soon enough.
